### ISSUE TITLE
Still having login issue with auth 404

### ISSUE THREAD

**Issue by nappyapp** [DIGITALCRIMINAL/OnlyFans#1434]
I realize with the last issue I misunderstood the auth_id number and that the script intentionally places the id in the other parameters. Sorry for bothering you.

However, I'm still having the issue with Auth 404'ed even after carefully following the instructions to set the script up, multiple times on multiple new installations of the script.

Here's my process, can you tell me if I'm doing something wrong? I'm using python 3.9.5

- git clone
- python -m pip install -r requirements.txt
- run the script
- I modify the config file to set "onlyfans" as the default site, and set browser auth to false (I've tried browser auth with username and password also, same problem)
- press enter on script
- edit .profile/default/auth.json. I fill in the username field with my username in quotes, then use the element inspector on the "init" request headers. I copy value of the cookie parameter and paste it in the cookie in the json file. I copy value of the user_agent parameter and paste it in user_agent in the json file.
- I leave email and password blank
- press enter on script
- script chooses onlyfans automatically, I press 1 to choose default profile
- long delay ensues, then Auth 404'ed each time until the script quits.

I know it's private info but IDGAF, I want your help so here's my auth.json:

![image](https://user-images.githubusercontent.com/79231864/124243885-378eeb80-dad3-11eb-9729-0edd6b69ecb1.png)

If I'm doing something wrong I'd appreciate you pointing it out, thanks!

Note on trying browser auth:
When I tried browser auth I set the browser auth option to true, and add my email and password to the config file. I do the rest of the steps the same. The script behaves in exactly the same way described already.

Here's my config.json:

```
{
  "info": {
    "version": 7.2
  },
  "settings": {
    "auto_site_choice": "onlyfans",
    "export_type": "json",
    "profile_directories": [
      ".profiles"
    ],
    "max_threads": -1,
    "min_drive_space": 0,
    "helpers": {
      "renamer": true,
      "reformat_media": true,
      "downloader": true,
      "delete_empty_directories": false
    },
    "webhooks": {
      "global_webhooks": [],
      "global_status": true,
      "auth_webhook": {
        "succeeded": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        },
        "failed": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        }
      },
      "download_webhook": {
        "succeeded": {
          "webhooks": [],
          "status": null,
          "hide_sensitive_info": true
        }
      }
    },
    "exit_on_completion": false,
    "infinite_loop": true,
    "loop_timeout": "0",
    "dynamic_rules_link": "https://raw.githubusercontent.com/DATAHOARDERS/dynamic-rules/main/onlyfans.json",
    "proxies": [],
    "cert": "",
    "random_string": "96c20939db0c11eb91b458961df7f377"
  },
  "supported": {
    "onlyfans": {
      "settings": {
        "auto_profile_choice": [],
        "auto_model_choice": false,
        "auto_media_choice": "",
        "auto_api_choice": true,
        "browser": {
          "auth": false
        },
        "jobs": {
          "scrape_names": true,
          "scrape_paid_content": true
        },
        "download_directories": [
          ".sites"
        ],
        "file_directory_format": "{site_name}\\{model_username}\\{api_type}\\{value}\\{media_type}",
        "filename_format": "{filename}.{ext}",
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}\\{model_username}\\Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklist_name": "",
        "webhook": true
      }
    },
    "starsavn": {
      "settings": {
        "auto_profile_choice": "",
        "auto_model_choice": false,
        "auto_media_choice": "",
        "auto_api_choice": true,
        "browser": {
          "auth": true
        },
        "jobs": {
          "scrape_names": true,
          "scrape_paid_content": true
        },
        "download_directories": [
          ".sites"
        ],
        "file_directory_format": "{site_name}\\{model_username}\\{api_type}\\{value}\\{media_type}",
        "filename_format": "{filename}.{ext}",
        "metadata_directories": [
          ".sites"
        ],
        "metadata_directory_format": "{site_name}\\{model_username}\\Metadata",
        "delete_legacy_metadata": false,
        "text_length": 255,
        "video_quality": "source",
        "overwrite_files": false,
        "date_format": "%d-%m-%Y",
        "ignored_keywords": [],
        "ignore_type": "",
        "blacklist_name": "",
        "webhook": true
      }
    }
  }
}
```

**Comment by DIGITALCRIMINAL** [comment]
> I realize with the last issue I misunderstood the auth_id number and that the script intentionally places the id in the other parameters. Sorry for bothering you.
> 
> However, I'm still having the issue with Auth 404'ed even after carefully following the instructions to set the script up, multiple times on multiple new installations of the script.
> 
> Here's my process, can you tell me if I'm doing something wrong? I'm using python 3.9.5
> 
> * git clone
> * python -m pip install -r requirements.txt
> * run the script
> * I modify the config file to set "onlyfans" as the default site, and set browser auth to false (I've tried browser auth with username and password also, same problem)
> * press enter on script
> * edit .profile/default/auth.json. I fill in the username field with my username in quotes, then use the element inspector on the "init" request headers. I copy value of the cookie parameter and paste it in the cookie in the json file. I copy value of the user_agent parameter and paste it in user_agent in the json file.
> * I leave email and password blank
> * press enter on script
> * script chooses onlyfans automatically, I press 1 to choose default profile
> * long delay ensues, then Auth 404'ed each time until the script quits.
> 
> I know it's private info but IDGAF, I want your help so here's my auth.json:
> 
> ![image](https://user-images.githubusercontent.com/79231864/124243885-378eeb80-dad3-11eb-9729-0edd6b69ecb1.png)
> 
> If I'm doing something wrong I'd appreciate you pointing it out, thanks!
> 
> Note on trying browser auth:
> When I tried browser auth I set the browser auth option to true, and add my email and password to the config file. I do the rest of the steps the same. The script behaves in exactly the same way described already.
> 
> Here's my config.json:
> 
> ```
> {
>   "info": {
>     "version": 7.2
>   },
>   "settings": {
>     "auto_site_choice": "onlyfans",
>     "export_type": "json",
>     "profile_directories": [
>       ".profiles"
>     ],
>     "max_threads": -1,
>     "min_drive_space": 0,
>     "helpers": {
>       "renamer": true,
>       "reformat_media": true,
>       "downloader": true,
>       "delete_empty_directories": false
>     },
>     "webhooks": {
>       "global_webhooks": [],
>       "global_status": true,
>       "auth_webhook": {
>         "succeeded": {
>           "webhooks": [],
>           "status": null,
>           "hide_sensitive_info": true
>         },
>         "failed": {
>           "webhooks": [],
>           "status": null,
>           "hide_sensitive_info": true
>         }
>       },
>       "download_webhook": {
>         "succeeded": {
>           "webhooks": [],
>           "status": null,
>           "hide_sensitive_info": true
>         }
>       }
>     },
>     "exit_on_completion": false,
>     "infinite_loop": true,
>     "loop_timeout": "0",
>     "dynamic_rules_link": "https://raw.githubusercontent.com/DATAHOARDERS/dynamic-rules/main/onlyfans.json",
>     "proxies": [],
>     "cert": "",
>     "random_string": "96c20939db0c11eb91b458961df7f377"
>   },
>   "supported": {
>     "onlyfans": {
>       "settings": {
>         "auto_profile_choice": [],
>         "auto_model_choice": false,
>         "auto_media_choice": "",
>         "auto_api_choice": true,
>         "browser": {
>           "auth": false
>         },
>         "jobs": {
>           "scrape_names": true,
>           "scrape_paid_content": true
>         },
>         "download_directories": [
>           ".sites"
>         ],
>         "file_directory_format": "{site_name}\\{model_username}\\{api_type}\\{value}\\{media_type}",
>         "filename_format": "{filename}.{ext}",
>         "metadata_directories": [
>           ".sites"
>         ],
>         "metadata_directory_format": "{site_name}\\{model_username}\\Metadata",
>         "delete_legacy_metadata": false,
>         "text_length": 255,
>         "video_quality": "source",
>         "overwrite_files": false,
>         "date_format": "%d-%m-%Y",
>         "ignored_keywords": [],
>         "ignore_type": "",
>         "blacklist_name": "",
>         "webhook": true
>       }
>     },
>     "starsavn": {
>       "settings": {
>         "auto_profile_choice": "",
>         "auto_model_choice": false,
>         "auto_media_choice": "",
>         "auto_api_choice": true,
>         "browser": {
>           "auth": true
>         },
>         "jobs": {
>           "scrape_names": true,
>           "scrape_paid_content": true
>         },
>         "download_directories": [
>           ".sites"
>         ],
>         "file_directory_format": "{site_name}\\{model_username}\\{api_type}\\{value}\\{media_type}",
>         "filename_format": "{filename}.{ext}",
>         "metadata_directories": [
>           ".sites"
>         ],
>         "metadata_directory_format": "{site_name}\\{model_username}\\Metadata",
>         "delete_legacy_metadata": false,
>         "text_length": 255,
>         "video_quality": "source",
>         "overwrite_files": false,
>         "date_format": "%d-%m-%Y",
>         "ignored_keywords": [],
>         "ignore_type": "",
>         "blacklist_name": "",
>         "webhook": true
>       }
>     }
>   }
> }
> ```

Hmm, do you have 2FA on? It might be because of that

**Comment by nappyapp** [comment]
Thanks for the reply. I have 2FA disabled on my onlyfans account, always have. And I've tried both 2FA enabled and disabled in the script config, same results as described above in both cases.

In the past it always worked fine having 2FA disabled on onlyfans site and enabled in the script config.



However, I just pulled the latest update to the script and it is now working again. Not sure what the problem was though, I'm pretty confused about it.
